examples/03_connectivity/plot_compare_decomposition.py

```diff
@@ -34,64 +40,103 @@
 print('First functional nifti image (4D) is at: %s' %
       rest_dataset.func[0])  # 4D data
 
-###############################################################################
-# Create two decomposition estimators
-# ------------------------------------
-from nilearn.decomposition import DictLearning, CanICA
 
-n_components = 40
-
-###############################################################################
-# Dictionary learning
-# --------------------
-#
+####################################################################
+# Here we apply CanICA on the data
+# ---------------------------------
 # We use as "template" as a strategy to compute the mask, as this leads
 # to slightly faster and more reproducible results. However, the images
 # need to be in MNI template space
-dict_learning = DictLearning(n_components=n_components,
+
+from nilearn.decomposition import CanICA
+
+canica = CanICA(n_components=20,
+                memory="nilearn_cache", memory_level=2,
+                threshold=3.,
+                n_init=1,
+                verbose=10,
+                mask_strategy='template',
```

Comment:
```suggestion
                mask_strategy='template',  # We use the MNI brain template to define the mask: faster and more reliable
```

Reply:
This is denoted in the text just above. Would you rather I delete the text above and represent the information as you have it here?

Reply:
You are right. I am not read the diff properly. Let me remove my suggestion.